gating/pre_merge_test/run_deploy.sh

```diff
@@ -48,7 +48,7 @@ if [[ ${RE_JOB_IMAGE} =~ no_artifacts$ ]]; then
   # Upgrade to the absolute latest
   # available packages.
   apt-get update
-  DEBIAN_FRONTEND=noninteractive apt-get -y upgrade
+  DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade
```

Comment:
Doing this upgrade may upgrade the kernel packages, but won't make them active. It requires a reboot to be active as far as I know. Is this what you want?

Reply:
Yeah, @cloudnull and I chatted and given there are some package dependencies on the kernel, this should still simulate the dependency tree without the reboot.